docs/extensions/conventions.md

```diff
@@ -6,6 +6,13 @@ Some of these extensions might need to create global resources in the seed (e.g.
 
 Consequently, this page should help answering some general questions that might come up when it comes to developing an extension.
 
+## Extension `Class`es
```

Comment:
Why the backticks? Class is not a resource. I'd remove them (also below in the description (except when talking about the `.spec.class` field